ratis-server/src/main/java/org/apache/ratis/server/impl/RaftConfigurationImpl.java

```diff
@@ -145,6 +145,17 @@ boolean containsInConf(RaftPeerId peerId) {
     return conf.contains(peerId);
   }
 
+  boolean isHighestPriority(RaftPeerId peerId) {
+    RaftPeer target = getPeer(peerId);
+    Collection<RaftPeer> peers = getCurrentPeers();
+    for (RaftPeer peer : peers) {
+      if (peer.getPriority() >= target.getPriority() && !peer.equals(target)) {
```

Comment:
Should it be `>`?

Reply:
@szetszwo For example, if server1, server2, server3 with the same priority, we can not transfer leader from server1 to server2,  because server1 only `yieldLeaderToHigherPriorityPeer ` when server1 find it's priorty is smaller than followers' priority, so server2 must has the highest priority.

Reply:
I see.  Thanks.